core/new-gui/src/app/dashboard/user/component/user-workflow/user-workflow.component.ts

```diff
@@ -278,7 +278,7 @@ export class UserWorkflowComponent implements OnInit, OnChanges {
       .subscribe(() => {
         if (this.userService.isLogin()) {
           this.refreshDashboardWorkflowEntries();
-          this.userProjectService.retrieveProjectList();
+          this.userProjectService.refreshProjectList();
```

Comment:
The returned observable is not used.

Reply:
The project list is used only in the `UserProjectListItem` component and not here. This is a place to refresh them on load.

Reply:
Since `refreshProjectList` is a public function, can we make it return void and call it in `UserProjectListItem` and use `retrieveProjectList` to get the observable?

Reply:
done